adapters/taboola/taboola.go

```diff
@@ -11,6 +11,10 @@ import (
 	"net/http"
 )
 
+const (
+	NATIVE_ENDPOINT = "https://native.bidder.taboola.com/OpenRTB/TaboolaHB/auction"
+)
```

Comment:
No need to hardcode it here. Modify endpoint URL in taboola.yaml file to 
`endpoint: "https://{{.MediaType}}.bidder.taboola.com/OpenRTB/TaboolaHB/auction"` and replace `MediaType` when you build request url. Similar to https://github.com/prebid/prebid-server/blob/91408f7d797dc7b425543a93e2eac8c4f5e228a4/adapters/adview/adview.go#L141

Reply:
@VeronikaSolovei9  Thanks for the great suggestion! just committed the changes using macros from now and on for building the endpoint url.

Reply:
Thank you for the change! Code looks so much cleaner now!